.pre-commit-config.yaml

```diff
@@ -98,13 +98,14 @@ repos:
   # Local checks for PEP headers and more
   - repo: local
     hooks:
-#      # Hook to run "check-peps.py"
-#      - id: "check-peps"
-#        name: "Check PEPs for metadata and content enforcement"
-#        entry: "python check-peps.py"
-#        language: "system"
-#        files: "^pep-\d{4}\.(rst|txt)$"
-#        require_serial: true
+      # Hook to run "check-peps.py"
+      - id: "check-peps"
+        name: "Check PEPs for metadata and content enforcement"
+        entry: "python check-peps.py"
+        language: "python"
```

Comment:
This means we're using the mapping above:

```yaml
default_language_version:
  python: python3
```

It works on macOS, and also passes on the CI for each OS, but let's confirm local use with a Windows user.

A downside is it creates a venv for it, but it will be cached, and I think is worth it to Windows and remove all the regexes.